### PR TITLE
0010: submit document as proposed

### PR DESCRIPTION
**Pull request by aesedepece**
I hereby ask for:
- WIP number assignment.
- Direct status of `Proposed`.

### CODE REVIEW

**Comment by lrubiorod** [comment]
I assign you the WIP-0010 number

File: wip-0010.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<pre>
+  WIP: WIP-0010
+  Title: Feb 2021 Chain Fork Post-Mortem
+  Author: The witnet-rust developers <devs@witnet.io>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0050
```
Comment (mariocao) [inline wip-0010.md:6]: Remove?

File: wip-0010.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+### Long term
+
+Beyond setting a minimum superblock committee size, these are some other ideas relevant to this issue that have suggested by different community members and are probably worth being further explored and discussed sooner or later:
+
+- Existence of a reliable "rescue committee" that can kick in and have "casting vote" rights when superblock consensus is lost. Identities could be added or removed from this committee through WIPs.
+- Incentivization for single superblock voting, either in form of rewards, penalties, or both.
+- Incentivization for nodes that accept inbound connections, so as to improve the ratio between public and private nodes.
+- A more convenient mechanism for automatically importing lists of peer addresses.
+- Some method for exporting and importing chain state snapshots, as a way to instantly getting the local chain into a particular block height.
```
Comment (mariocao) [inline wip-0010.md:82]:
```suggestion
- Some method for exporting and importing chain state snapshots, as a way to instantly get the local chain into a particular block height.
```

Comment (aesedepece) [reply wip-0010.md:81]: I'm a maximalist of this rule of thumb:
- to + infinitive
- of + gerund
- for + gerund

With the only exception of a few phrasal verbs, e.g. "look forward to", "be used to", "be committed to", "be devoted to", etc.

**Review by lrubiorod — APPROVED** [review]

**Pull request merged**